packages/wells/src/client/model/Survey.ts

```diff
@@ -0,0 +1,42 @@
+import { SequenceRow } from 'stable/dist/src/api/sequences/sequenceRow';
+
+export type SearchSurveys = (args?: any) => Promise<Survey[]>;
+
+export interface Survey {
```

Comment:
Things like assetId or externalId are characteristics of the internal representation (Sequence) and should not be on the Survey data class. 

The survey should instead contain a direct reference to its Wellbore (as in the Wellbore data class). I expect we will add more attributes to the Survey data class once Javeir and Julie finish the mapping of data, but the wellbore itself contains a lot of the info that helps people working with the survey, for example, for visualization. 

So, with a direct reference, we would have something like thisSurvey.wellbore.well.waterDepth. We can add aliases as we proposed between wells and wellbores, so if the user calls, for example,  thisSurvey.getWaterDepth(), we would just return the attribute from thisSurvey.wellbore.well.waterDepth. 
In any case, having the reference to the wellbore directly fetched when we fetch the survey would allow us to do that without having to make another call to CDF to find the asset with that assetId and then do the same repeating CDF calls up through the hierarchy until we find the well

Reply:
Should we do this in a separate PR mabye?

Reply:
I can remove assetId and externalId from the Survey data class in this PR